src/rust/cryptography-x509-validation/src/lib.rs

```diff
@@ -10,3 +10,14 @@ pub mod ops;
 pub mod policy;
 pub mod trust_store;
 pub mod types;
+
+#[derive(Debug, PartialEq, Eq)]
+pub enum ValidationError {
+    Other(String),
```

Comment:
NB: Owned rather than `&'static` here because some errors in #8873 will ultimately be "rendered," e.g. including OIDs for critical extensions that aren't handled.